src/services/currency-conversion.js

```diff
@@ -25,9 +25,19 @@ const addDecimalToNumber = number => {
   return `${dollars}.${cents}`
 }
 
+const checkForNegative = number => {
```

Comment:
instead of a check & add stage, would a conditional `-` check on the regex (line: 17) achieve the same thing? seems a little simpler